### ISSUE TITLE
Add sklearn.preprocessing.LabelEncoder

### ISSUE THREAD

**Issue by yujonglee** [repllabs/fastrepl#39]
We need to make it easy to use [sklearn.preprocessing.LabelEncoder](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html), or implement it for pipeline interface.

Need some good warning if labels is not encoded. See `test_model_grade.py`
